app/components/page-components/Chatbot/module-bases/ChatbotAlertsBase.vue.ts

```diff
@@ -0,0 +1,91 @@
+import { cloneDeep } from 'lodash';
+import { Component, Prop } from 'vue-property-decorator';
+import ChatbotWindowsBase from 'components/page-components/Chatbot/windows/ChatbotWindowsBase.vue';
+import {
+  IChatAlertsResponse
+} from 'services/chatbot/chatbot-interfaces';
+import { debug } from 'util';
+
+@Component({})
+export default class ChatbotAlertsBase extends ChatbotWindowsBase {
+  get chatAlerts() {
+    return this.chatbotApiService.state.chatAlertsResponse;
+  }
+
+  get chatAlertsEnabled() {
+    return this.chatbotApiService.state.chatAlertsResponse.enabled;
+  }
+
+  get alertTypes() {
+    const { streamlabs, twitch } = this.chatAlerts.settings;
+    let alertTypes = {
+      ...streamlabs,
+      ...twitch
+    }
+    delete alertTypes.bits;
+    return alertTypes;
+  }
+
+  platformForAlertType(type: string) {
+    if (type === 'tip') return 'streamlabs';
+    return 'twitch';
+  }
+
+
+  // preparing data to send to service
+
+  // update/delete alert
+  spliceAlertMessages(
+    type: string,
+    index: number,
+    updatedAlert: any,
```

Comment:
This should be `IAlertMessage` right?  Best to avoid using `any` whenever possible.